spec/takeaway_spec.rb

```diff
@@ -1,14 +1,23 @@
 require "takeaway"
 
 describe Takeaway do
-    subject(:takeaway) { described_class.new(menu: menu) }
+  subject(:takeaway) { described_class.new(menu: menu, order: order) }
 
-    let(:menu) { double(:menu, print: menu_output) }
-    let(:menu_output) { "Meal: Price" }
+  let(:menu) { double(:menu, print: menu_output) }
+  let(:order) { double(:order) }
+  let(:menu_output) { "Meal: Price" }
 
-    context 'menu' do
-        it 'shows the menu including dishes and prices' do
-        expect(takeaway.provide_menu).to eq(menu_output)
-        end
+  let(:dishes) { {burger: 2, vege: 1} }
+
+  context 'menu' do
+    it 'shows the menu including dishes and prices' do
+      expect(takeaway.provide_menu).to eq(menu_output)
     end
-end
\ No newline at end of file
+  end
+
+  it 'can order some dishes' do
+    expect(order).to receive(:add).twice
```

Comment:
not familiar with the "twice" method, but I'm guessing you might want to add dishes three times for example, would it still work for that purpose?